e2e/libs/workload/statefulset.py

```diff
@@ -10,7 +10,7 @@
 from utility.utility import logging
 
 
-def create_statefulset(statefulset_name, volume_type, option):
+def create_statefulset(statefulset_name, volume_type, sc_name):
     filepath = "./templates/workload/statefulset.yaml"
```

Comment:
For this change, do we need to add `sc_name` the default value, similar to what was done in https://github.com/longhorn/longhorn-tests/pull/1904/files#diff-cb5abaea0420a63451de0d96179c6a7859721fccec1cbdcd80940c8920eba665R34?

Reply:
Not necessary. `create_statefulset` in `statefulset.py` is called by `create_statefulset` in `statefulset_keywords.py` which already specifies the default value. You can reference `create_volume` in `volume_keywords.py` which uses the same strategy.